{
  "contest": 372,
  "handle": "cu5t0mpeo",
  "risk": "3",
  "title": "When the contract is paused, the withdraw function can still work normally.",
  "issueId": 296,
  "issueUrl": "https://github.com/code-423n4/2024-04-renzo-validation/issues/296",
  "validatedIssueId": 128,
  "validatedIssueUrl": "https://github.com/code-423n4/2024-04-renzo-findings/issues/128"
}